src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,4 +1,5 @@
 *REMOVED*abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.MemberAccessExpression(Microsoft.CodeAnalysis.SyntaxNode expression, Microsoft.CodeAnalysis.SyntaxNode memberName) -> Microsoft.CodeAnalysis.SyntaxNode
+*REMOVED*Microsoft.CodeAnalysis.Workspace.RaiseDocumentActiveContextChangedEventAsync(Microsoft.CodeAnalysis.Document document) -> System.Threading.Tasks.Task
```

Comment:
This proposal changes RaiseDocumentActiveContextChangedEventAsync, which is very unlikely to be used by anybody. If we have to keep the old overload, I'm not exactly sure how we'll be able to populate all the new information.